src/steps/cloudwaf/converter.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  createIntegrationEntity,
+  createDirectRelationship,
+  parseTimePropertyValue,
+  Entity,
+  RelationshipClass,
+  Relationship,
+} from '@jupiterone/integration-sdk-core';
+
+import { Entities } from '../constants';
+import { SignalSciencesCloudWAF } from '../../types';
+
+export function getCloudWAFKey(id: string): string {
+  return `sigsci_cloudwaf:${id}`;
+}
+
+export function createCloudWAFEntity(cloudWAF: SignalSciencesCloudWAF): Entity {
+  return createIntegrationEntity({
+    entityData: {
+      source: cloudWAF,
+      assign: {
+        _type: Entities.CLOUD_WAF._type,
+        _class: Entities.CLOUD_WAF._class,
+        _key: getCloudWAFKey(cloudWAF.id),
+        name: cloudWAF.name,
+        displayName: cloudWAF.name,
+        category: ['application'],
+        description: cloudWAF.description,
+        region: cloudWAF.region,
+        tlsMinVersion: cloudWAF.tlsMinVersion,
+        siteName: cloudWAF.workspaceConfigs[0].siteName,
+        'deployment.Status': cloudWAF.deployment.status,
```

Comment:
Is there are reason to keep the `.`? Why not deploymentStatus?

Reply:
This is something suggested by @austinkelleher long time ago, we had quite a few scenarios where response object contained nested objects and one idea was to keep the original response's structure by using `.` to designate the "nesting level".

The `deploymentStatus` could work just as well, I think it just comes down to preference and patterns (if there is any concrete one for cases such as this one - would be great if there is one).

Reply:
I haven't made any of these changes just yet, but could easily if we decide we want that pattern instead.